flows/codelingo/review/review/cli.go

```diff
@@ -166,6 +167,13 @@ func reviewAction(cliCtx *cli.Context) (chan proto.Message, <-chan *flowutil.Use
 		return nil, nil, nil, nil, errors.Trace(err)
 
 	}
+	wd, err := os.Getwd()
+	if err != nil {
+		return nil, nil, nil, nil, errors.Trace(err)
+
+	}
+	parts := strings.Split(wd, "/")
+	path := parts[2] + "/" + parts[len(parts)-1]
```

Comment:
I don't think this will work on a machine where `GOPATH` is `/home/users/me/go`, for example; you're assuming `GOPATH` is a certain length. You should explicitly get the go path and trim it from the working directory. If there is no `GOPATH` set then `path` should be `""`. This will make the purpose of the code much clearer too.

Reply:
Sorry, I misunderstood what you were trying to do. Trying to get the ubuntu username won't work as per our conversation:
```suggestion
	path := parts[len(parts)-2] + "/" + parts[len(parts)-1]
```

Reply:
However, reviews don't have to run from the root of a repo. So you'll actually have to use the git root rather than the working directory.

The fact that `path` should still be empty when you don't have a `GOPATH` variable still stands. Path only applies to go.